src/NuGet.Clients/NuGet.SolutionRestoreManager/VsSolutionRestoreService.cs

```diff
@@ -76,6 +77,21 @@ public sealed class VsSolutionRestoreService : IVsSolutionRestoreService
 
         public Task<bool> CurrentRestoreOperation => _restoreWorker.CurrentRestoreOperation;
 
+        public Task<bool> NominateProjectAsync(string projectUniqueName, CancellationToken token)
+        {
+            if (string.IsNullOrEmpty(projectUniqueName))
```

Comment:
I looked it up. It seems that we are not really using it anywhere else, any reason particularly? Are we going to transition to using it going forward? Just to keep consistency I would keep it the way it is, but if you feel that `Assumes` is the way to go, I can change it. Thoughts?